src/Onboarding/Routes/SettingsRoute.php

```diff
@@ -4,16 +4,43 @@
 
 use WP_REST_Request;
 use Give\API\RestRoute;
+use Give\Onboarding\Helpers\Currency;
+use Give\Onboarding\SettingsRepository;
 
 class SettingsRoute implements RestRoute {
 
-	protected $endpoint = 'onboarding/settings';
+	protected $endpoint = 'onboarding/settings/(?P<setting>\w+)';
```

Comment:
I updated the endpoint structure to dynamically accept the setting name in the URL, so that a specific setting can implement specific handling without implementing an entirely unique route.

See CurrencyRoute for an example implementation.